packages/unit-testing/README.md

```diff
@@ -32,14 +34,243 @@ module.exports = SuiteCloudJestConfiguration.build({
+      > Suitelet.js
+		> Templates
+		> Web Site Hosting Files
+	> Objects
+	> Translations
+  deploy.xml
+  manifest.xml
+suitecloud.config.js
+jest.config.js
+```
+
+To see the content of the files you need to make SuiteCloud Unit Testing work, check the following list:
+
+- `jest.config.js` file
+```javascript
+const SuiteCloudJestConfiguration = require("@oracle/suitecloud-unit-testing/jest-configuration/SuiteCloudJestConfiguration");
+
+module.exports = SuiteCloudJestConfiguration.build({
+	projectFolder: 'src',
+	projectType: SuiteCloudJestConfiguration.ProjectType.ACP,
+});
+```
+
+- `Suitelet.js` file
+
+```javascript
+/**
+ * @NApiVersion 2.x
+ * @NScriptType Suitelet
+ * @NModuleScope SameAccount
+ */
+define(["N/record"], function(record) {
+    return {
+        onRequest: function(context) {
+            if (context.request.method === 'GET') {
+                const salesOrderId = context.request.parameters.salesOrderId;
+                let salesOrderRecord = record.load({id: salesOrderId});
+                salesOrderRecord.setValue({fieldId: 'memo', value: "foobar"});
+                salesOrderRecord.save({enableSourcing: false});
+            }
+        }
+    };
+});
+```
+
+- `Suitelet.test.js` file
+
+```javascript
+import Suitelet from "SuiteScripts/Suitelet";
+
+import record from "N/record";
+import Record from "N/record/instance";
+
+jest.mock("N/record");
+jest.mock("N/record/instance");
+
+beforeEach(() => {
+    jest.clearAllMocks();
+});
+
+describe("Suitelet Test", () => {
+    it("Sales Order memo field has been updated", () => {
+        // given
+        const context = {
+            request: {
+                method: 'GET',
+                parameters: {
+                    salesOrderId: 1352
+                }
+            }
+        };
+
+        record.load.mockReturnValue(Record);
+        Record.save.mockReturnValue(1352);
+
+        // when
+        Suitelet.onRequest(context);
+
+        // then
+        expect(record.load).toHaveBeenCalledWith({id: 1352});
+        expect(Record.setValue).toHaveBeenCalledWith({fieldId: 'memo', value: 'foobar'});
+        expect(Record.save).toHaveBeenCalledWith({enableSourcing: false});
+    });
+});
+
+```
+
+To read a comprehensive description of this example, visit [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_158436693543.html#subsect_158437822806).
+
+### Custom Stub Example 
+
+In order for your tests to fully work, your project needs to follow a specific structure. Use the following structure as an example:
+
+```
+myAccountCustomizationProject
+> customStubs
+	> http.js
+> __tests__
+	> http.test.js
+> src
+	> AccountConfiguration
+	> FileCabinet
+		> SuiteScripts
+		> Templates
+		> Web Site Hosting Files
+	> Objects
+	> Translations
+	deploy.xml
+	manifest.xml
+suitecloud.config.js
+jest.config.js
+```
+
+Once you have a file structure similar to the one above, you can start creating the testing files. 
```

Comment:
I would remove this line too, the test file is already in the structure above: http.test.js